ledger-service/http-json/src/main/scala/com/digitalasset/http/PackageService.scala

```diff
@@ -311,7 +311,7 @@ object PackageService {
       interface: iface.Interface
   ): Map[(TemplateId.RequiredPkg, Choice), iface.Type] =
     interface.typeDecls.flatMap {
-      case (qn, iface.InterfaceType.Template(_, iface.DefTemplate(choices, _, _))) =>
+      case (qn, iface.InterfaceType.Template(_, iface.DefTemplate(choices, _, _, _))) =>
```

Comment:
I'm not sure how `getChoices` is used, but keep in mind that this implementation will not return any choices inherited from interfaces

Reply:
ah, ignore me, I see now that resolved interface choices get appended to the main list by `resolveChoices`